static/solo-cta/index.js

```diff
@@ -280,21 +279,52 @@ function handleModalInitialized(e){
           runTime: clickData.runTime
       }, '*');
   }
-
-  if (utils.isIOS) {
-    var onOrientationChange = function () {
-        if (iframeModal && iframeModal.contentWindow) {
-            var width = iframeModal.parentNode.offsetWidth;
-            iframeModal.contentWindow.window.postMessage({
-                event: config.eventPrefix + ':modal_holder_resize',
-                size: [width, Math.floor(width * (9 / 16))]
-            },'*');
+  var detectOrientation = function(){
```

Comment:
I really need to understand what issues are being tackled here, thanks.
Ticket will work otherwise please explain.

Reply:
Sounds related?
https://tvpage.atlassian.net/browse/DEV-7876

Reply:
@luisvelazquex can you help here, i think this was you right?

Reply:
@luisvelazquex I think we need to revisit this fix. Please remove this code from the branch.
There's a potential performance issue in the code, I explain this below.